src/com/pentalog/pentastagiu/homework2_bookCandyShop/Basket.java

```diff
@@ -0,0 +1,94 @@
+package com.pentalog.pentastagiu.homework2_bookCandyShop;
+
+public class Basket {
+    private Double totalPrice=0.00;
+    private Customer customer;
+    private Product product;
```

Comment:
cred ca ar fi mai simplu sa ai o lista de produse care au fost adaugate in cos(cantitatea poti sa o ignori pentru moment si doar cand e clara implementarea cu o lista simpla de produse sa adaugi si implementarea cu cantitati) iar cele 2 metode in care calculezi discountul si pretul sa nu primeasca nimic ca parametru, ci pur si simplu sa lucreze cu lista de produse (de ex sa adune pretul tuturor produselor din cos si abia apoi sa aplici logica actuala)